cmd/hologram-agent/main.go

```diff
@@ -62,10 +64,18 @@ func main() {
 		log.Debug("Using command-line remote address.")
 		config.Host = *dialAddress
 	}
+	if *accessKey != "" {
```

Comment:
I think I'm leaning towards using the provider chain in aws-sdk-go, since it's a canonical approach that's used in other language sdk's too. Plus we could always drop a hint in error logs or a usage message (e.g. "create this symlink" or do whatever you need to).